tools/cldr-apps/src/main/java/org/unicode/cldr/web/CalendarSortMode.java

```diff
@@ -220,31 +220,33 @@ public boolean isMember(DataRow p) {
     }
 
     @Override
-    Comparator<DataRow> createComparator() {
-        return comparator();
+    Comparator<DataRow> getComparator() {
+        return COMPARATOR;
     }
 
-    public static Comparator<DataRow> comparator() {
+    static final Comparator<DataRow> COMPARATOR = new  Comparator<DataRow>() {
```

Comment:
This doesn't appear right. The key to the Bill Pugh singletons is that the statics are in their own static class. That keeps them from being created until needed.

Reply:
The `SortMode` subclasses are intertwined, many sibling classes make use of the Comparator instance from another sibling. But, I can make these lazily instantiated.